packages/react-magma-dom/src/components/ToggleButtonGroup/ToggleButtonGroup.test.js

```diff
@@ -0,0 +1,133 @@
+        </ToggleButtonGroup>
+      );
+      const wrapper = getByTestId(testId);
+
+      expect(wrapper).toHaveStyleRule('row-gap', magma.spaceScale.spacing03);
+      expect(getByTestId(`${testId}-1`)).toHaveStyleRule(
+        'padding',
+        `${magma.spaceScale.spacing04} ${magma.spaceScale.spacing05}`
+      );
+    });
+  });
+
+  describe('States', () => {
+    // it('Should allow just one selected button at a time', () => {
+    //   const { getByText } = render(
+    //     <ToggleButtonGroup exclusive>
+    //       <ToggleButton>Button One</ToggleButton>
+    //       <ToggleButton>Button Two</ToggleButton>
+    //     </ToggleButtonGroup>
+    //   );
+
+    //   const buttonOne = getByText('Button One').parentElement;
+
+    //   fireEvent.click(buttonOne);
+
+    //   expect(buttonOne.firstChild).toHaveStyleRule(
+    //     'background',
+    //     transparentize(0.5, magma.colors.neutral300)
+    //   );
+    // });
+
+    it('Should allow multiple selected buttons at a time', () => {
+      const { getByTestId } = render(
+        <ToggleButtonGroup>
+          <ToggleButton testId={testId}>{TEXT}</ToggleButton>
+          <ToggleButton testId={`${testId}-1`}>{TEXT}</ToggleButton>
+        </ToggleButtonGroup>
+      );
+
+      fireEvent.click(getByTestId(testId));
+
+      expect(getByTestId(testId)).toHaveStyleRule(
+        'background',
+        transparentize(0.5, magma.colors.neutral300)
+      );
+
+      fireEvent.click(getByTestId(`${testId}-1`));
+
+      expect(getByTestId(testId)).toHaveStyleRule(
+        'background',
+        transparentize(0.5, magma.colors.neutral300)
+      );
+      expect(getByTestId(`${testId}-1`)).toHaveStyleRule(
+        'background',
+        transparentize(0.5, magma.colors.neutral300)
+      );
+    });
+
+    it('Should allow a default selection set by the user', () => {
+      const { getByTestId } = render(
+        <ToggleButtonGroup value="one">
+          <ToggleButton value="one" testId={testId}>
+            {TEXT}
+          </ToggleButton>
+          <ToggleButton testId={`${testId}-1`}>{TEXT}</ToggleButton>
+        </ToggleButtonGroup>
+      );
+
+      expect(getByTestId(testId)).toHaveStyleRule(
+        'background',
+        transparentize(0.5, magma.colors.neutral300)
+      );
+    });
+
+    it('Should allow a default selection set by the user to be unselected', () => {
+      const { getByTestId } = render(
+        <ToggleButtonGroup value="one">
+          <ToggleButton value="one" testId={testId}>
+            {TEXT}
+          </ToggleButton>
+          <ToggleButton testId={`${testId}-1`}>{TEXT}</ToggleButton>
+        </ToggleButtonGroup>
+      );
+
+      expect(getByTestId(testId)).toHaveStyleRule(
+        'background',
+        transparentize(0.5, magma.colors.neutral300)
+      );
+
+      fireEvent.click(getByTestId(testId));
+
+      expect(getByTestId(testId)).toHaveStyleRule('background', 'none');
+    });
+  });
+});
```

Comment:
Can we add a few more tests:
- check that `onChange` gets called
- verify `enforced` prop
- if we change `value` to a string, we should test with 1 value and then with multiple